pipeline_dp/private_beam.py

```diff
@@ -308,3 +315,117 @@ def __init__(self, fn: Callable, label: Optional[str] = None):
     def expand(self, pcol: pvalue.PCollection):
         return pcol | "flatten values" >> beam.ParDo(
             FlatMap._FlattenValues(map_fn=self._fn))
+
+
+class PrivateCombineFn(beam.CombineFn):
+
+    @abc.abstractmethod
+    def add_private_input(self, accumulator, input):
+        pass
+
+    @abc.abstractmethod
+    def extract_private_output(self, accumulator):
```

Comment:
ditto, could we add some some instructions for developers who will be implementing this method?